src/features/dashboard/Dashboard.tsx

```diff
@@ -4,16 +4,12 @@ import {IntegrationContext} from "../../context/IntegrationContext";
 import DashboardCard from "./DashboardCard";
 import {ICard} from "./types/Card";
 import {useTranslation} from "react-i18next";
-import {DashboardStyles} from "../../util/styles/Dashboard.styles";
 import {IIntegrationStatistics} from "./types/IntegrationStatistics";
 import PageTemplate from "../../components/templates/PageTemplate";
 import {RouteComponent} from "../../routes/Route";
 
-const useStyles = DashboardStyles;
-
 const Dashboard: RouteComponent = () => {
     const {t} = useTranslation('translations', {keyPrefix: 'pages.dashboard'});
```

Comment:
Jeg har gjort endringer på Dashboard og DashboardCard i FFS-649 - vil du dra den branchen min inni denne er det helt ok . hvis ikke tar jeg evt mergconf etter du har deploya denne

Reply:
Jeg har gjort så få endringer at jeg ser for meg det blir en "ok" conflict å rette opp i, det blir vel mest å bare beholde dine endringer tenker jeg?